### DISCUSSION TITLE
Stuck at "There was an error trying to log you in: 'No account to get tokens for.'"

### DISCUSSION THREAD

**Discussion by heyvoon** [FBoucher/TinyBlazorAdmin#61] [Q&A]
Hi All,

I did all the steps but found myself stuck at the following error when trying to login.

There was an error trying to log you in: 'No account to get tokens for.'

![image](https://user-images.githubusercontent.com/3281265/112039167-07cf5f80-8b44-11eb-8b0e-d137dafaf631.png)

App Registration
![image](https://user-images.githubusercontent.com/3281265/112039277-29304b80-8b44-11eb-8689-905ec878d982.png)

Resource Group
![image](https://user-images.githubusercontent.com/3281265/112039397-49f8a100-8b44-11eb-87c6-b28692eba3d7.png)

Any idea what I could have done wrong?

UPDATE: Somehow I pass the erros above but now I am stuck at;

![image](https://user-images.githubusercontent.com/3281265/112110453-32a4cc80-8bb3-11eb-8dd2-2074438771d6.png)

Didn't change much!

I've tried to install everything from scratch 3 times now following all the steps.
In all 3 installations I went through all the settings again at least 3 times looking for mistakes. Nothing.
Still fails! am I doing really something wrong or thing is made that is not suppose to work because of some flaw in the tutorial?

**Comment by fboucher** [comment]
I'm sorry to read that @heyvoon  I did a lot of cleaning/ merging/ moving things around... Maybe I deleted something. I will do a pass see what's wrong.

**Reply by ghost** [reply]
I am on the exact same step and stuck. Any guidance would be appreciated. 
![tinyblazor1](https://user-images.githubusercontent.com/81591319/112907252-28069d80-90b3-11eb-9d52-00d3b766cdd4.png)

**Reply by heyvoon** [reply]
Hi @hpashamedpt 

That are a lot of issues during the deployment because the documentation steps are very unclear. I was able to pass some of the errors I described above. But I still could never fully deploy as expected.

1. Logout page doesn't work! Page not found!
2. It is NOT clear whether we have to create one or two Static App!
2. It is NOT clear where you have put to configuration of the custom domain in the settings.json

I followed all the steps to configured custom domain but it still doesn't work! (Screenshot)

![image](https://user-images.githubusercontent.com/3281265/112967671-cba38c80-914b-11eb-8098-fefb337b56c9.png)
The URL shortner works only if I do not use the custom domain but still fails when loging out!

It's a shame that the documentation is really confused. This is such a nice project. It just doesn't work as expected. I guess you need to be a really pro to be able to figure this out!
**I have tried to start the installation all over again at least 4 times but the results are still the same.**

**Reply by NUIAZ** [reply]
same here

**Comment by heyvoon** [comment]
I also would like to point out that after deployment, I keep getting this non stop!

![image](https://user-images.githubusercontent.com/3281265/112972955-1b388700-9151-11eb-9162-5bd98c04d50f.png)

**Comment by TheJollyRoger2022** [comment]
Hello.  I am getting either 404 or function does not exist in Tenant ID.  I have deployed 4 times and modified .json as many ways as I can think of.  I end up with the same errors.  This would be a great tool if it would work.  Has something changed in Azure AD to stop this from working?  Can you give me any other suggestions?  Do you have a video of you deploying and configuring this tool step by step?  Thanks for any additional assistance.
